packages/toolkit/src/query/tests/queryLifecycle.test.tsx

```diff
@@ -53,7 +54,7 @@ describe.each([['query'], ['mutation']] as const)(
             async onQueryStarted(arg, { queryFulfilled }) {
               onStart(arg)
               // awaiting without catching like this would result in an `unhandledRejection` exception if there was an error
-              // unfortunately we cannot test for that in jest.
+              // unfortunately we cannot test for that in vi.
```

Comment:
💁 

this seems like a victim of search and replace - it's probably better to refer to it as `vitest` here rather than `vi`